java/bulk-import/bulk-import-runner/src/test/java/sleeper/bulkimport/job/runner/dataframe/AddPartitionIteratorTest.java

```diff
@@ -50,9 +50,9 @@ public void shouldAddPartitionField() {
         Iterator<Row> rows = Arrays.asList(row1, row2).iterator();
         List<Object> splitPoints = Collections.singletonList(3);
         PartitionsFromSplitPoints partitionsFromSplitPoints = new PartitionsFromSplitPoints(schema, splitPoints);
-        PartitionTree partitionTree = new PartitionTree(schema, partitionsFromSplitPoints.construct());
-        String partition1 = partitionTree.getLeafPartition(Key.create(1)).getId();
-        String partition2 = partitionTree.getLeafPartition(Key.create(4)).getId();
+        PartitionTree partitionTree = new PartitionTree(partitionsFromSplitPoints.construct());
+        String partition1 = partitionTree.getLeafPartition(schema, Key.create(1)).getId();
+        String partition2 = partitionTree.getLeafPartition(schema, Key.create(4)).getId();
```

Comment:
This is duplicating the implementation a bit. If we use PartitionsBuilder we could assert the partitions explicitly. I know it was already like this but it seems worth changing.

Reply:
Thanks, updated this to use `PartitionsBuilder` and assert on partition IDs explicitly